runtime/src/accounts_db.rs

```diff
@@ -285,6 +286,13 @@ impl<'a> LoadedAccount<'a> {
             },
         }
     }
+
+    pub fn from_cache(&self) -> bool {
```

Comment:
nit: `from_` prefixes in Rust generally imply the argument is consumed and transformed into another type. How about `is_cached()`?